frontend/src/metabase/parameters/components/widgets/TemporalUnitWidget/TemporalUnitWidget.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Button } from "metabase/ui";
+import * as Lib from "metabase-lib";
+
+const MIN_WIDTH = 180;
+
+interface TemporalUnitWidgetProps {
+  setValue: (unit: string) => void;
+  onClose: () => void;
+}
+
+export function TemporalUnitWidget({
+  setValue,
+  onClose,
+}: TemporalUnitWidgetProps) {
+  const units = Lib.availableTemporalUnits();
+
+  const handleSelect = (unit: string) => {
+    setValue(unit);
+    onClose();
+  };
+
+  return (
+    <Box p="sm" miw={MIN_WIDTH}>
```

Comment:
- no item background on hover (see `timeseries-bucket-button` for comparison)
- only text is interactive, not the entire item (see `timeseries-bucket-button` for comparison)
- in "Default value" input, the dropdown is shifted horizontally from the trigger

https://github.com/metabase/metabase/assets/6830683/08294ba6-4337-40f4-8f3f-888384050224